src/components/Navbar/index.tsx

```
@@ -282,69 +203,72 @@ const Navbar = () => {
-                              return;
-                            }
-
-                            const ci = new CONTRACT(
-                              TOKEN_WVOI,
-                              getAlgorandClients().algodClient,
-                              getAlgorandClients().indexerClient,
-                              abi.nt200,
-                              {
-                                addr: activeAccount.address,
-                                sk: new Uint8Array(0),
+                      {balanceData && balanceData.success ? (
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)